src/planner/binder/expression/bind_window_expression.cpp

```diff
@@ -192,6 +192,10 @@ BindResult SelectBinder::BindWindow(WindowExpression &window, idx_t depth) {
 	unique_ptr<FunctionData> bind_info;
 	if (window.type == ExpressionType::WINDOW_AGGREGATE) {
 		//  Look up the aggregate function in the catalog
+		if (window.catalog.empty() && window.schema.empty()) {
+			window.catalog = SYSTEM_CATALOG;
+			window.schema = DEFAULT_SCHEMA;
+		}
 		auto func = Catalog::GetEntry<AggregateFunctionCatalogEntry>(context, window.catalog, window.schema,
 		                                                             window.function_name, false, error_context);
 		D_ASSERT(func->type == CatalogType::AGGREGATE_FUNCTION_ENTRY);
```

Comment:
Perhaps this should be an exception instead of an assertion?

Reply:
In Catalog::GetEntry an error is thrown if the function is not of type aggregate 
https://github.com/duckdb/duckdb/blob/master/src/include/duckdb/catalog/catalog.hpp#L213